### ISSUE TITLE
Code with specified language should render as fenced codeblocks

### ISSUE THREAD

**Issue by marijnvdwerf** [thephpleague/html-to-markdown#70]
**Input**:
```html
<pre><code class="language-ruby">def foo(x)
  return 3
end
</code></pre>
```

**Actual**:
```md
    <code class="language-ruby">def foo(x)
      return 3
    end
```

**Expected**:
````md
```ruby
def foo(x)
  return 3
end
``` # comment after end fence because GitHub is weird
````

Also, care should be taken that fences are longer than any candidate-fences in the block

**Comment by quarkmarino** [comment]
+1

**Comment by andreskrey** [comment]
#102 fixes this. Check it out if you can :)

**Comment by colinodell** [comment]
Fixed via #102
